website/src/media/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "instantsearch-media",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "rm -rf ./dist/ && rsync -r . ./dist/"
```

Comment:
what's the reason for using rsync instead of cp here, is it faster?

We don't have an issue not supporting windows now, but that might cause issues later. For now I guess it's fine either way

Reply:
It worked well in that case. Those build scripts are temporary and should be run only via Netlify anyway.